search/src/Components/wrapper/upload.md

```diff
@@ -1,13 +1,20 @@
 ## Annotation Formats
 
-We currently support uploads in MS COCO and [WeedCOCO](/weedcoco) formats.
+Annotations can currently be uploaded in the following formats, for which we note the supported annotation layers:
 
+| Format | Classification | Bounding Boxes | Segmentation |
+|:-------|:--------------:|:--------------:|:--------------:|
+| [WeedCOCO](./weedcoco) | ✔️ | ✔️ | ✔️ |
+| [MS COCO](https://cocodataset.org/#format-data) | ✔️ | ✔️ | ✔️ |
+| Pascal VOC XML | | ✔️ | |
+| Color-coded segmentation masks | | | ✔️ |
 
 We natively support WeedCOCO format which extends on MS COCO to specify a weed
 ID-oriented category naming scheme, to include agricultural context and
-[schema.org/Dataset](https://schema.org/Dataset)-compatible metadata. We provide an uploader
-for MS COCO format, with forms to enter agricultural context and metadata
-please ensure the category names are conformant before uploading.
+[schema.org/Dataset](https://schema.org/Dataset)-compatible metadata.
+Uploads in other formats be converted to WeedCOCO format, with
+forms to enter agricultural context and metadata, as well as to
```

Comment:
Probably worth linking those here: [agricultural context](https://weed-ai.sydney.edu.au/editor) and [metadata](https://weed-ai.sydney.edu.au/meta-editor)